pages/linux/fpsync.md

```diff
@@ -0,0 +1,28 @@
+# fpsync
+
+> Execute several synchronization processes locally or on several remote workers through SSH.
+> More information: <https://www.fpart.org/fpsync/>.
+
+- Recursivly syncronize a directory to another location:
+
+`fpsync -v {{/absolute/path/to/source/}} {{/absolute/path/to/destination/}}`
```

Comment:
```suggestion
`fpsync -v {{/path/to/source/}} {{/path/to/destination/}}`
```
I don't think we need to say that the path is absolute, the `/` before is enough.